src/main/auth/index.ts

```diff
@@ -26,8 +26,9 @@ export const getUserDetails = async (
   const secret: string = config.get('services.idam.clientSecret');
   const tokenUrl: string = config.get('services.idam.tokenURL');
   const callbackUrl = encodeURI(serviceUrl + callbackUrlPageLink);
-  const code = encodeURIComponent(rawCode);
-  const data = `client_id=${id}&client_secret=${secret}&grant_type=authorization_code&redirect_uri=${callbackUrl}&code=${code}`;
+  // const code = encodeURIComponent(rawCode);
```

Comment:
Did you mean to commit this? I thought this is just a local change.

Reply:
Fixed that